src/Connectors/KurrentDB.Connectors.Tests/ClusterVNodeApp.cs

```diff
@@ -54,6 +55,9 @@ public class ClusterVNodeApp : IAsyncDisposable {
             .With(x => x.Services.AddSingleton<IHostedService>(esdb))
             .With(x => configureServices?.Invoke(x.Services));
 
+        if(useRandomPort)
+			builder.WebHost.ConfigureKestrel(serverOptions => serverOptions.ListenAnyIP(Random.Shared.Next(5001, 6000)));
```

Comment:
Needed not to have conflicts on an open port for the cluster vNode. I didn't pass a specific port, as for now, it's not needed.

Reply:
there is a way to make bind to some available port i think, possibly specifying port 0?